data/human/human missions.txt

```diff
@@ -5428,3 +5428,71 @@ mission "Deal Change 2"
 				die
 			label skip
 			`	After a few minutes, the pirates finish unloading the cargo, and Ci steps forward, holding out a credit chip. "<payment>, for making the right choice." Ci and the pirates return to their ships as you take off and head towards the main spaceport of <planet>.`
+
+
+
+mission "Quicksilver Mixup [2]"
+	name "Misposted mail delivery"
+	description "Carry misposted mail from Silver to Quicksilver by <date> and then come back with the return shipment"
+	minor
+	deadline 10 1
+	source "Silver"
+	destination "Quicksilver"
+	cargo "Misposted mail" 38
+	to offer
+		has "Quicksilver Mixup [1]: done"
+		has "quicksilver mixup 2 available"
+	on offer
+		conversation
+			`As you walk through Silver's spaceport, somebody wearing office attire power walks up to you after ending a hurried conversation with another pilot.`
+			`	"Captain! I work for the postmaster's office. The regular pilot who carries misposted mail to Quicksilver and back couldn't make it this month, some mechanical issues with his ship, I'm told. Would you be willing to carry this month's load there by <day>? We'll pay you what he'd be paid, 100.000 credits."`
+			choice
+				`	"Oh! So you guys did set up that regular shipment, after all!"`
+					goto context
+
+				`	"Sure, how much mail are we talking about?"`
+					goto shipment
+
+				`	"Sorry, but i'm not heading in that direction"`
+					goto refuse
+
+			label context
+			`	The mail employee appears a tad confused for a moment, but then you mention you once delivered a misposted package like that, and the clerk had mentioned somebody should set up a regular delivery due to how common of an issue it was. The post employee smiles and responds: "Oh! That was me, actually, back when I worked as a clerk for the post office. I was sick of always having that same conversation with so many pilots a day, so I went out and found a captain to set up a regular delivery with and made the proposal to my boss, who was just as sick of my and the Quicksilver's postmaster office endless reports. Got a bonus and promotion for it - I'm now responsible for the delivery's management, instead! Anyways, are you interested in covering for them?"`
+			choice
+				`	"Yeah, what's the size of the shipment?"`
+					goto shipment
+				`	"Sorry, but I'm not able to work that into my plans."`
+					goto refuse
+
+			label shipment
+			`	The man checks his notes and says "This week, it totalled... 38 tons of letters and miscellaneous packages here, and then 29 tons to be brought back from Quicksilver."`
```

Comment:
Is "checks his notes" cliché in English? I didn't know. 

Personally, I find "flips through" to imply disorganisation, like somebody hurriedly throwing around the pages of a notebook, and with this guy being a seemingly organised clerk-turned-manager, it doesn't fit imo. I am fine with changing it to something else neutral, that imply his notes are easily accessible, just requiring him to find the specific line with the info again. 

What do you think of "The man takes a look at his tablet and says/says,"? It could be added to the start of the mission, as well ("somebody wearing office attire **and carrying a tablet** power walks up...")

About the comma, I'm fine with it either way.

Reply:
Checks notes is a weird joke in English I've never got my head around. The tablet idea sounds good though.

The comma is necessary anyway : )

```suggestion
			`	The man takes a look at his tablet and says, "This week, it totalled... 38 tons of letters and miscellaneous packages here, and then 29 tons to be brought back from Quicksilver."`
```

Reply:
Neat! I'll commit it and add the tablet line to the start of the mission, too!